getting-started/go/README.MD

```diff
@@ -1 +1 @@
-[![Contribute](https://www.eclipse.org/che/contribute.svg)](https://che.openshift.io/f?url=https://raw.githubusercontent.com/redhat-developer/devfile/master/getting-started/go/devfile.yaml)
+[![Contribute](https://www.eclipse.org/che/contribute.svg)](https://workspaces.openshift.com/f?url=https://raw.githubusercontent.com/redhat-developer/devfile/master/getting-started/go/devfile.yaml)
```

Comment:
operator.apps.sandbox.x8i5.p1.openshiftapps.com/plugin-registry/v3/plugins/ms-vscode/go/latest/meta.yaml failed. Status code 404. Response body:



Not Found

The requested URL was not found on this server.

Reply:
@ericwill is it expected that `ms-vscode` is not available in CRW ?

Reply:
This plugin hasn't been available in upstream Che or CRW for awhile now:

https://github.com/redhat-developer/codeready-workspaces/blob/crw-2-rhel-8/dependencies/che-devfile-registry/devfiles/05_go/devfile.yaml

https://github.com/eclipse/che-devfile-registry/blob/master/devfiles/go/devfile.yaml

Reply:
@ericwill thanks for the input. I see it is replaced with `golang/go/latest`